index.test-d.ts

```diff
@@ -16,3 +16,5 @@ expectType<string>(
 	})
 );
 expectType<string>(replaceString(input, '🐑', '🦄', {fromIndex: 1}));
+expectType<string>(replaceString(input, '🐑', '🦄', {caseInsensitive: true as boolean}));
```

Comment:
Why `as boolean`?

Reply:
So that in only one line of test we assert that `caseInsensitive` can take any boolean, not just `true`.

I almost did this at first:

```ts
expectType<string>(replaceString(input, '🐑', '🦄', {caseInsensitive: true}));
expectType<string>(replaceString(input, '🐑', '🦄', {caseInsensitive: false}));
```

But that felt exaggerated to me. So I thought of this `true as boolean` compromise.